ops/model.py

```diff
@@ -913,6 +913,13 @@ def __init__(self, relation: 'Relation', entity: 'UnitOrApplication',
         self._backend = backend
         self._is_app = isinstance(entity, Application)  # type: bool
 
+    @property
+    def _strict_access_control(self) -> bool:
```

Comment:
Semantically, this name seems like it should be something more like "_hook_is_running".

Reply:
fair enough, renaming the underlying call seems bad